pkg/watch/health.go

```diff
@@ -36,6 +36,7 @@ const TTL = 60 * time.Second
 // has a running MonitorHealth go routine
 type PodWatch struct {
 	manifest pods.Manifest
+	client   *http.Client
```

Comment:
I want to use the same http client for health checks that is used by everything else in the preparer. I thought about adding methods to the Store interface for actually performing health checks but that didn't really make sense to me.